STYLE.md

```diff
@@ -0,0 +1,81 @@
+
```

Comment:
It's a bit meta to have a markdown nit in a review of a Python style guide, but could we get rid of the blank line? None of the other .md files have a leading blank line, and I think it's better not there.

Reply:
Fixed